paper-toggle-button.html

```diff
@@ -134,12 +93,29 @@
         transition: none;
       }
 
+      :host([checked]):not([disabled]) .toggle-bar {
+        opacity: 0.5;
+        background-color: var(--paper-toggle-button-checked-bar-color, --default-primary-color);
+        @apply(--paper-toggle-button-checked-bar);
+      }
+
+      :host([disabled]) .toggle-bar {
+        background-color: #000;
+        opacity: 0.12;
+      }
+
       :host([checked]) .toggle-button {
         -webkit-transform: translate(16px, 0);
         transform: translate(16px, 0);
       }
 
+      :host([checked]):not([disabled]) .toggle-button {
+        background-color: var(--paper-toggle-button-checked-button-color, --default-primary-color);
+        @apply(--paper-toggle-button-checked-button);
+      }
+
       :host([disabled]) .toggle-button {
+        /* Need !important to override the value set in `:host([checked]) .toggle-button` */
```

Comment:
that actually didn't work :(
[`disabled` +` checked`] is the same as just [`checked`], so i didn't do anything (and only picked up the `checked` style -- which is why i had to split that one into [`checked` and `not disabled`]

>.<